dm/templates/project/project.py

```diff
@@ -105,7 +113,7 @@ def generate_config(context):
     }
 
 
-def activate_apis(properties):
+def activate_apis(context, properties):
```

Comment:
Context contains properties. I would not change the input params for activate_apis, but keep it for context and create a local properties variable in the beginning of the function.